cookieconsent/templates/includes/banner.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+
+{% if show_banner %}
+<div class="wrapper wrapper--cookieconsent cookieconsent no-js-hidden">
+    <div class="wrapper wrapper--small">
+        <h2 class="cookieconsent__title">{% trans title %}</h2>
+        <div class="cookieconsent__message">
+            {% trans message|safe %}
```

Comment:
It would be good to use [|richtext](https://docs.wagtail.io/en/v2.8.1/topics/writing_templates.html#rich-text-filter) filter.